src/pilot/boot_mode_helper.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
+
+# Copyright (c) 2018 Dell Inc. or its subsidiaries.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ironic.drivers.modules import deploy_utils
+
+DRAC_BOOT_MODE_BIOS = 'Bios'
+DRAC_BOOT_MODE_UEFI = 'Uefi'
+
+DRAC_TO_IRONIC_BOOT_MODES = {
+    DRAC_BOOT_MODE_BIOS: deploy_utils.SUPPORTED_CAPABILITIES['boot_mode'][0],
+    DRAC_BOOT_MODE_UEFI: deploy_utils.SUPPORTED_CAPABILITIES['boot_mode'][1]
+}
+
+IRONIC_TO_DRAC_BOOT_MODES = \
```

Comment:
Similar to the above comment, please change the name of the dictionary to

    IRONIC_BOOT_MODE_CAP_TO_DRAC_BOOT_MODE

A longer option is `IRONIC_BOOT_MODE_CAPABILITY_TO_DRAC_BOOT_MODE`.

Regardless of what is chosen, the names of the two (2) dictionaries should be consistent.